src/components/userProfile/UserProfile.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import {
+    StyleSheet,
+    View,
+    Text,
+    ScrollView,
+    Image,
+    TouchableHighlight,
+    Dimensions,
+    ListView,
+} from 'react-native';
+import { Actions, } from 'react-native-router-flux';
```

Comment:
Mostly, it's required to place a comma on import or some other parts even if there's nothing after a comma. However, it seems okay without a comma on that part.